### ISSUE TITLE
missing treesitter dependencies on fedora

### ISSUE THREAD

**Issue by kabessao** [LunarVim/LunarVim#2195]
### Feature Description

While installing Lunarvim on my Fedora 35 i ran into some issues. Luckily I just needed to install some dependencies. So it would help new users on Fedora to have those dependencies listed on the README.txt. 

Here are the dependencies I had to install to make it work: 
libstdc++.x86_64
libstdc++-devel.x86_64
libstdc++-static.x86_64
g++

### Describe the alternatives you have considered

Having the install.sh script check if those dependencies are met, and giving an error to the user if they are not found.

**Comment by Uzaaft** [comment]
An idea could be to run:
```bash
sbin/ldconfig -p | grep stdc++
```
And use the output to see if the library is installed. Unless I am attacking this in a wrong way?

**Comment by kabessao** [comment]
I've uninstalled these packages and reinstalled lvim fresh and I was able to replicate the error. 
Here is a print showing the error:
![image](https://user-images.githubusercontent.com/22626848/150662439-c08be95f-e694-4050-a9c1-4a1f24a62586.png)

After installing g++ it worked. This package has the libstdc++-static.x86_64 as it's dependency, so it gets installed together.

Just installing libstdc++-static.x86_64 brings this error:
![image](https://user-images.githubusercontent.com/22626848/150662604-566808b6-ec44-4334-942b-7b34825d8213.png)

Finally, the command ```sbin/ldconfig -p | grep stdc++``` brings the same output before and after installing all the packages
